web/styles/settings.css

```diff
@@ -770,9 +770,18 @@ input[type="checkbox"] {
     float: right;
 
     #show-add-default-streams-modal {
-        padding: 5px 0;
-        min-width: 100px;
-        margin-top: 11px;
+        /* We want this button to have the same height as the filter
+           button on its right. We're overriding the default vertical
+           padding of `.new-style` and setting it to 0. Having the
+           height fixed will mean that the vertical padding will adjust
```

Comment:
I am somewhat confused by this line. The padding will still remain 0 irrespective of the font size.

Reply:
I meant to refer to the whitespace above and below the font, I've replaced it to say `whitespace`, does that make the comment more clear?

Reply:
Yes, that makes it more clear.